collections/ansible_collections/purestorage/flashblade/plugins/modules/purefb_s3user.py

```diff
@@ -63,6 +63,14 @@
     - Access key secret for access key to import
     type: str
     version_added: "1.4.0"
+  policy:
+    description:
+    - User Access Policies to be assigned to user on creation
+    - To amend policies use the I(purefb_userpolicy) module
+    - If not specified, I(pure\:policy/full-access) will be added
```

Comment:
Remove line 70 and add following:

```yaml
    default: policy/full-access
```

Reply:
Actually, the APIs will create that default policy for a user without needing to set it - hence why I need to actually remove it if not specified in the `policy` parmaeter

Reply:
That makes sense then.